docs/src/main/sphinx/release/release-434.md

```diff
@@ -0,0 +1,114 @@
+# Release 434 (29 Nov 2023)
+
+## General
+
+* Add support for a `FILTER` clause to the `LISTAGG` function. ({issue}`19869`)
+* {{breaking}} Rename the `query.max-writer-tasks-count` configuration property
+  and the related `max_writer_tasks_count` session property to
+  `query.max-writer-task-count` and `max_writer_task_count`. ({issue}`19793`)
+* Improve performance of `INSERT ... SELECT` queries that contain a redundant
+  `ORDER BY` clause. ({issue}`19916`)
+* Fix incorrect results for queries involving comparisons between `double` and
+  `real` zero and negative zero. ({issue}`19828`)
+* Fix performance regression caused by suboptimal scalar subqueries planning. ({issue}`19922`)
+* Prevent table function failure when its invocation is a first query run on a
+  worker. ({issue}`19849`)
+* Prevent sudden increases in memory consumption in some queries using
```

Comment:
Typo in "JOINT".

Also, there's no such a thing as CROSS JOIN UNNEST. Cross join and unnest are two separate constructs. Therefore, I would rephrase it as "join involving UNNEST".

Reply:
Updated